components/jupyter-web-app/backend/kubeflow_jupyter/common/utils.py

```diff
@@ -24,6 +25,8 @@
 EVENT_TYPE_NORMAL = "Normal"
 EVENT_TYPE_WARNING = "Warning"
 
+EVENT_ERROR_MSG_REGEX = re.compile(
```

Comment:
Should we change the `Insufficient cpu` to just `Insufficient` in order to also include other cases such as not enough memory or gpus?

Reply:
See comment 
https://github.com/kubeflow/kubeflow/issues/4705#issuecomment-582396315

I think we should focus on surfacing events rather than trying to correctly classify them. 

Insufficient resources could be a permanent error e.g. user requests 32 CPUs and only has 16 CPU machines.

Or it could be temporary, because of autoscaling or just because some other workloads will finish allowing it to be scheduled.